### PR TITLE
Add Account Endpoint and Logic

### PR DESCRIPTION
**Pull request by MichaelCduBois**
This is a work in progress. Items I need to complete:
- [x] `accounts/:id/verify` logic
- [x] `accounts/:id/verify` returns JWT if successful

`accounts` Sends email if existing configuration -- Will be completed with email integration.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,38 @@
...
 
 This server is written in [Go][go-website] using the [Echo][echo-website] framework.
 
+## Local Development
+
+Leagueify API uses easy to use Makefile commands to get up and running quickly. Currently, Leagueify API requires Go 1.22.0 in order to run, please install Go before proceeding with the following commands:
+
+``` bash
+# Install Dependencies
+make init
```
Comment (kylekrny) [inline README.md:13]: When I attempt to run make init I get the error:
```
go get .
make: go: No such file or directory
make: *** [init] Error 1
```

**Review by kylekrny — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MichaelCduBois) [reply README.md:13]: Might be a dumb question...is Go installed?

**Comment by kylekrny** [comment]
@MichaelCduBois If I hit server.localhost I should get a response right? API docs should be there?

**Comment by MichaelCduBois** [comment]
> @MichaelCduBois If I hit server.localhost I should get a response right? API docs should be there?

server.localhost/api

I'll update the README.

**Review by kylekrny — APPROVED** [review]

**Pull request merged**